packages/ast-spec/tests/fixtures.test.ts

```diff
@@ -72,7 +72,7 @@ const FIXTURES: readonly Fixture[] = [...VALID_FIXTURES, ...ERROR_FIXTURES].map(
         }
       })(),
       ext,
-      isError: absolute.includes('/_error_/'),
+      isError: /[\\/]_error_[\\/]/.test(absolute),
```

Comment:
what's the difference between these two? Seem like they're the same?

Reply:
No, windows has backslash and Linux/Mac/Unix has slash

Reply:
@bradzacher is there anything blocking this change?

Reply:
ah - i wasn't paying attention - it's now a regex literal so the `\\` means `\` - it's not escaping the backslash to escape the `/`.